src/server/services/hli/hli-zorgned-service.ts

```diff
@@ -116,19 +115,21 @@ export const fetchNamenBetrokkenen = memoizee(fetchNamenBetrokkenen_, {
   maxAge: 45 * ONE_SECOND_MS,
 });
 
-function assignIsActueel(aanvraagTransformed: ZorgnedAanvraagTransformed) {
+function isActueel(aanvraagTransformed: ZorgnedAanvraagTransformed) {
   const isEOG = isEindeGeldigheidVerstreken(aanvraagTransformed);
   let isActueel = aanvraagTransformed.isActueel;
 
-  // Override actueel indien de einde geldigheid is verlopen
-  if (isActueel && isEOG) {
-    isActueel = false;
-  }
-  if (!isActueel && !isEOG) {
-    isActueel = true;
+  switch (true) {
+    case !aanvraagTransformed.datumIngangGeldigheid:
+      return false;
+    // Override actueel indien de einde geldigheid is verlopen
+    case isActueel && isEOG:
```

Comment:
Deze conditie kan nu weg volgens mij door die `return false` aan het einde

Reply:
Dat is niet hetzelfde, EOG wordt gechecked adhv datum**Einde**Geldigheid. Case op 123 is datum**Ingang**Geldigheid

Reply:
Ah! thanks